hack/build-images.sh

```diff
@@ -16,9 +16,10 @@ if [ "${RELEASE_STREAM:-}" = 'prod' ] ; then
 fi
 dockerfile="Dockerfile${docker_suffix}"
 
+name_suf="5"
```

Comment:
so do we still want the version 5 images published to e.g. logging-elasticsearch5 instead of logging-elasticsearch?